hkmc2/shared/src/test/mlscript/nofib/treejoin.mls

```diff
@@ -0,0 +1,81 @@
848 4232 8088|20985 1201 4329|16570 17815 13983|16110 15523 2622|4241 10316 9889|5496 24688 1013|1326 1326 17003|5455 25866 25738|22853 22616 2328|10404 15684 9868|7170 12858 9223|14899 18398 16763|18393 3465 5372|9373 29 7693|5014 23454 13118|20311 16159 26842|19592 1336 23701|5180 1337 969|879 5879 7218|26486 1339 19142|16785 1340 22964|14397 6613 26640|17659 17894 9230|12991 21335 10906|8192 22101 15333|1364 9257 17489|17527 21306 114|24206 26142 4558|24460 13364 23657|2901 9669 16280|190 10675 5755|12679 1351 4602|4784 9424 8965|11657 1353 6716|7026 26623 15602|13635 5395 19443|26732 1356 3745|12120 1016 24488|2203 24990 19763|22274 17039 15415|19109 12344 15912|8444 13929 3068|17106 1362 10767|13990 24198 26414|9028 26580 14833|21653 8432 19800|851 14606 9838|9210 26882 22159|2461 7037 23680|5356 19697 1369|14050 8234 8111|18422 22606 14987|18060 7588 20820|16805 17112 21336|7654 622 20246|16215 22183 17442|1376 10989 1376|14092 21876 13348|13991 8306 26007|7502 26003 14766|20924 1380 7137|6149 26645 8757|7414 2918 25574|10402 18495 22330|18301 1384 22613|13561 26993 17956|1386 24802 24234|203 18734 1870|10124 15124 10036|25245 4405 5973|5395 8262 963|4431 1391 13047|17653 1392 9728|6756 2241 13617|14167 14930 1394|24102 16387 4662|11572 3801 18844|26600 16517 20045|22275 646 24595|15042 7327 21759|13944 19728 2072|19588 1401 24105|14338 15730 22327|14859 17763 9646|11052 9380 81|15789 976 12365|12203 23523 16662|5306 1407 17594|17157 26400 22160|13241 26097 1201|19151 9154 22042|534 19910 21590|26577 1276 16772|15341 6781 13237|2206 1414 2910|7959 3282 13146|15181 7232 17277|9292 4044 5300|19706 1298 22514|3142 1859 18070|22324 8676 21937|24664 25640 23912|7483 4398 1422|19215 19442 22818|19672 1424 4592|3369 817 21604|4223 17770 6866|8286 19022 21427|6185 22697 23484|26304 3973 21085|16318 10974 25171|12535 15786 14623|1432 6269 12461|15964 652 1353|11183 17378 14039|18294 1435 17411|4732 1436 11900|9200 9981 15368|2899 25483 9363|17591 13978 7506|26549 7672 26717|7441 12180 21633|15002 20578 7231|22 18606 23947|22953 1028 3900|5448 26168 24120|21355 3083 18819|4167 8986 6703|2925 7856 944|9780 13556 9284|25439 1450 4082|9758 1451 16878|9953 6452 1852|1453 5696 20024|14918 22638 14667|18983 9527 6327|4160 8589 7317|281 21460 16308|25634 8138 20767|14078 9086 11835|21673 11809 21001|13981 24424 16496|9451 4955 11118|20175 8618 5455|22736 2405 8989|22321 26300 17889|23914 23426 24575|20598 12774 3742|5065 23257 18201|1165 16760 21224|17779 1470 10915|8714 25079 6458|20365 1213 6760|948 3641 12012|22247 22367 450|14659 20038 2574|25116 12433 23121|18648 11232 9688|24926 14275 12934|12919 22375 10439|11064 9760 12096|7708 19396 545|18506 6575 14578|17838 14414 3211|22129 21844 17169|13408 24120 20021|6843 10166 2475|9087 6338 15679|8381 8597 1080|13596 14105 25076|26247 1490 7626|2190 12494 19427|15505 6025 13724|19909 8480 5416|7403 1219 20294|6570 1759 4474|5368 11080 15808|12281 1497 26401|20895 18287 7647|2403 16566 6934|17393 1500 14217")
```

Comment:
Are these huge string literals given in the source like this in nofib?

Reply:
Not really, they are read from files. Maybe I can write some utility function in prelude to read files and use them to get these big string literals?

Reply:
It should be really easy to read things from a file using the NodeJS API. There's an example of writing `Accounting.mls`. So I don't think it really needs helper functions (but if it does please don't put them in `Predef`; put them in an `IO` module).

Reply:
Thanks! I will refer to that to read the input from files.

Reply:
Now `mate.mls`, `rsa.mls`, `sorting.mls` and `treejoin.mls` read from files. Indeed no helper function is needed😃